### ISSUE TITLE
Storage add-ons for shared hosting.

### ISSUE THREAD

**Issue by rmehta** [frappe/press#1536]
"Storage is cheap" - @adityahase 

Anecdotally this has been a source of churn.

**Comment by NagariaHussain** [comment]
CFBR 😄

**Comment by NagariaHussain** [comment]
It will be database storage add-on btw. Actual "storage" is usually unused ...

**Comment by rmehta** [comment]
DB "storage" is way more expensive (it is proportional to memory as well) - maybe differential rates? My use case was for files.

**Comment by NagariaHussain** [comment]
Yeah, rates can be different but DB add ons have been requested by users way more than files in my observation.
